modules/qtwidgets/properties/transferfunctioneditor.h

```diff
@@ -62,38 +63,35 @@ class IVW_MODULE_QTWIDGETS_API TransferFunctionEditor : public QGraphicsScene {
     TransferFunctionEditor(TransferFunctionProperty* tfProperty, QWidget* parent = nullptr);
     virtual ~TransferFunctionEditor();
 
-    float getZoomRangeXMin() const;
-    void setZoomRangeXMin(float min);
-    float getZoomRangeXMax() const;
-    void setZoomRangeXMax(float max);
-    float getZoomRangeYMin() const;
-    void setZoomRangeYMin(float min);
 
-    float getZoomRangeYMax() const;
-    void setZoomRangeYMax(float max);
+    virtual void onControlPointAdded(TFPrimitive* p);
```

Comment:
Override?

Reply:
No, the editor is not yet an observer. Matching function names by "coincidence", I guess.